src/components/FileUploader/FileUploader.js

```diff
@@ -115,17 +172,22 @@ export class FileUploaderButton extends Component {
 
 export class Filename extends Component {
   static propTypes = {
+    /**
+     * Specify an optional object of styles to be applied inline to the root
+     * node
+     */
     style: PropTypes.object,
+
+    /**
+     * Specify the status of the File Upload
+     */
     status: PropTypes.oneOf(['edit', 'complete', 'uploading']),
-    tabIndex: PropTypes.number,
-    onKeyDown: PropTypes.func,
   };
 
   static defaultProps = {
     onKeyDown: () => {},
     status: 'uploading',
     style: {},
-    tabIndex: 0,
```

Comment:
This prop seems to be included in `…other` portion. Changing default prop here seems to affect there?

Reply:
Good call!